src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15106,13 +15121,15 @@ namespace UnitarySystems {
                             HeatingCoils::SimulateHeatingCoilComponents(state,
                                                                         CompName,
                                                                         FirstHVACIteration,
-                                                                        DataLoopNode::SensedLoadFlagValue,
+                                                                        this->m_DesignSuppHeatingCapacity,
                                                                         CompIndex,
                                                                         QCoilActual,
                                                                         SuppHeatingCoilFlag,
                                                                         FanOpMode,
                                                                         PartLoadFrac);
-                            PartLoadFrac = QCoilActual / this->m_DesignSuppHeatingCapacity;
+                            if (this->m_DesignSuppHeatingCapacity > 0.0) {
+                                PartLoadFrac = QCoilActual / this->m_DesignSuppHeatingCapacity;
+                            }
```

Comment:
if design supp heat capacity = 0 then PartLoadFrac = 1 from above at line 15116. That would actually seem correct (i.e., tried to run coil full out).

Reply:
Here also, PartLoadFrac = 1 so there is no need to calculate it.

Reply:
> Here also, PartLoadFrac = 1 so there is no need to calculate it.

Is it possible that it could deliver less than `DesignSuppHeatingCapacity` so calculating PLR might be useful?

Reply:
Other than a high temperature limit on the child I don't think so but's it's also hard to know if this calculation was placed here for a reason. Leaving as is for further evaluation.